tests/roles/dataplane_adoption/tasks/main.yaml

```diff
@@ -166,6 +166,8 @@
             - configure-os
             - run-os
           env:
+            - name: ANSIBLE_CALLBACKS_ENABLED
+              value: "profile_tasks"
```

Comment:
looks fine but trying to find the 'enhanced' output there 

https://logserver.rdoproject.org/84/84/429dd133c1b41fc84e86ce00f79c843d08716342/github-check/centos-9-standalone-to-crc-data-plane-adoption/42c2a60/controller/data-plane-adoption-tests-repo/data-plane-adoption/tests/logs/test_minimal_out_2023-08-02T13:14:46EDT.log

looks same to me or i am missing something?

```
TASK [dataplane_adoption : deploy dataplane] ***********************************
changed: [localhost] => {"changed": true, "cmd": "set -euxo pipefail\n\n\noc apply -f - <<EOF\napiVersion: dataplane.openstack.org/v1beta1\nkind: OpenStackDataPlane\nmetadata:\n  name: openstack\nspec:\n  deployStrategy:\n    deploy: true\n  nodes:\n
```

eg compared to another run https://logserver.rdoproject.org/75/75/2c2b5544ffd85946282f4bf147e036f48ad44afe/github-check/data-plane-adoption-github-rdo-centos-9-crc-single-node/6a8a7c9/controller/data-plane-adoption-tests-repo/data-plane-adoption/tests/logs/test_minimal_out_2023-08-02T18:45:32EDT.log

Reply:
@fao89 can you please rebase for merge see below:
```
This branch is out-of-date with the base branch
```

there is a button there i wonder if that works (the "update with rebase" option maybe?)

Reply:
I've rebased, and the change is only for AEE pods: https://logserver.rdoproject.org/84/84/69b1a7837e6681af224433762421756d2bd93d10/github-check/data-plane-adoption-github-rdo-centos-9-crc-single-node/9f030cd/controller/pod/dataplane-deployment-configure-os-edpmadoption-j6tqg-logs.txt